core/server/worker/src/test/java/alluxio/worker/block/DefaultBlockWorkerTest.java

```diff
@@ -212,11 +213,28 @@ public void getReport() {
   }
 
   @Test
-  public void getStoreMeta() {
+  public void getStoreMeta() throws Exception {
     mBlockWorker.getStoreMeta();
-    mBlockWorker.getStoreMetaFull();
     verify(mBlockStore, times(2)).getBlockStoreMeta(); // 1 is called at metrics registration
-    verify(mBlockStore).getBlockStoreMetaFull();
+    long blockId1 = mRandom.nextLong();
+    long blockId2 = mRandom.nextLong();
+    long sessionId = mRandom.nextLong();
+    mBlockWorker.createBlock(sessionId, blockId1, 0, "", 1L);
+    // Available space in `MEM` is not enough. Block will be allocated in `HDD`.
```

Comment:
That's not true. It's related to the `put(PropertyKey.WORKER_MANAGEMENT_TIER_ALIGN_RESERVED_BYTES, "0")` which  if reserved bytes is bigger than mem tier size, the blocks will not be allocated in the mem tier

Reply:
I check in `DefaultStorageDir` -> `mAvailableBytes = capacityBytes - reservedBytes` ,
available space means that the reserved space is removed. allocator will judge whether there is enough space based on AvailableBytes. so i think this description is right...